ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_fft.py

```diff
@@ -0,0 +1,40 @@
+# global
+from hypothesis import strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# fft
+@handle_frontend_test(
+    fn_tree="jax.numpy.fft.fft",
+    dtype_values_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        num_arrays=1,
+        shape=(2,),
+        min_axis=-1,
+        force_int_axis=True,
+    ),
+    n=st.integers(min_value=2, max_value=10),
+    norm=st.sampled_from(["backward", "ortho", "forward"]),
```

Comment:
```suggestion
    norm=st.sampled_from(["backward", "ortho", "forward", None]),
```
We should also test for `norm=None`